cirq/ops/display_test.py

```diff
@@ -199,6 +235,21 @@ def test_approx_pauli_string_expectation_value(measurements, value):
     assert display.value_derived_from_samples(measurements) == value
 
 
+@pytest.mark.parametrize('measurements, value, coefficient', [
+    (np.array([[0, 0, 0], [0, 0, 0]]), 1, 0.123),
+    (np.array([[0, 0, 0], [0, 0, 1]]), 0, 999),
+    (np.array([[0, 1, 0], [1, 0, 0]]), -1, -1),
+    (np.array([[0, 1, 0], [1, 1, 1]]), -1, 1),
+])
+def test_approx_pauli_string_expectation_value_with_coef(
+        measurements, value, coefficient):
+    display = cirq.pauli_string_expectation(cirq.PauliString(
+        {}, coefficient=coefficient),
```

Comment:
wonky formatting

Reply:
```
$ check/format-incremental 
Comparing against revision 'origin/master'.

Changes in cirq/ops/display_test.py require formatting:
--- cirq/ops/display_test.py    (original)
+++ cirq/ops/display_test.py    (reformatted)
@@ -243,8 +243,9 @@
 ])
 def test_approx_pauli_string_expectation_value_with_coef(
         measurements, value, coefficient):
-    display = cirq.pauli_string_expectation(
-        cirq.PauliString({}, coefficient=coefficient), num_samples=1)
+    display = cirq.pauli_string_expectation(cirq.PauliString(
+        {}, coefficient=coefficient),
+                                            num_samples=1)
     assert display.value_derived_from_samples(measurements) == \
            value * coefficient

```

Reply:
The formatter always knows best, even when it doesn't.